CRM/Core/xml/Menu/Api4.xml

```diff
@@ -3,6 +3,7 @@
   <item>
     <path>civicrm/ajax/api4</path>
     <page_callback>CRM_Api4_Page_AJAX</page_callback>
+    <access_callback>CRM_Api4_Permission::check</access_callback>
     <access_arguments>access CiviCRM;access AJAX API</access_arguments>
```

Comment:
Since the `access_callback` doesn't make use of the `access_arguments`, perhaps it'll confuse future-readers (who might edit it and find no behavioral change) -- perhaps remove it and/or replace with an explanatory comment?

Reply:
+1 to remove it